packages/nimble-components/src/theme-provider/design-tokens.ts

```diff
@@ -503,6 +503,12 @@ export const spinnerLargeHeight = DesignToken.create<string>(
     styleNameFromTokenName(tokenNames.spinnerLargeHeight)
 ).withDefault('64px');
 
+// The token gets a default value of empty-string but is a value in
+// the table styles.
+export const tableFitToRowsHeight = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.tableFitToRowsHeight)
+).withDefault('');
```

Comment:
Maybe could set it to the table's default height (480px) just so it's a real value. Don't feel strongly.